pkgs/tools/system/hw-probe/default.nix

```diff
@@ -0,0 +1,108 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+
+# Required
+, hwinfo
+, dmidecode
+, smartmontools
+, pciutils
+, usbutils
+, edid-decode
+
+# Recommended
+, withRecommended ? false # Install recommended tools
+, mcelog
+, hdparm
+, systemd
+, acpica-tools
+, drm_info
+, mesa-demos
+, memtester
+, sysstat
+, cpuid
+, util-linuxMinimal
+, xinput
+, libva-utils
+, inxi
+, vulkan-utils
+, i2c-tools
+, opensc
+
+# Suggested
+, withSuggested ? false # Install (most) suggested tools
+, hplip
+, sane-backends
+, pnputils ? null # pnputils (lspnp) isn't current in nixpkgs and appears to be poorly maintained
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hw-probe";
+  version = "1.6.4";
+
+  src = fetchFromGitHub {
+    owner = "linuxhw";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:028wnhrbn10lfxwmcpzdbz67ygldimv7z1k1bm64ggclykvg5aim";
+  };
+
+  makeFlags = [ "prefix=$(out)" ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  makeWrapperArgs =
+    let
+      requiredPrograms = [
+        hwinfo
+        dmidecode
+        smartmontools
+        pciutils
+        usbutils
+        edid-decode
+      ];
+      recommendedPrograms = [
+        mcelog
+        hdparm
+        systemd # (systemd-analyze)
+        acpica-tools
+        drm_info
+        mesa-demos
+        memtester
+        sysstat # (iostat)
+        cpuid
+        util-linuxMinimal # (rfkill)
```

Comment:
I've used `util-linuxMinimal` instead of util-linux to copy something similar in inxi. Not sure what best practice is

Reply:
util-linuxMinimal is probably just for bootstrapping systemd and related friends. Shouldn't matter ot much which we use if minimal has the features compiled in.